.github/workflows/create-release-draft.yml

```diff
@@ -173,13 +173,9 @@ jobs:
     steps:
       - name: Checkout codes on ${{ env.RELEASE_TAG }}
         uses: actions/checkout@v4
-        with:
-          ref: ${{ env.RELEASE_TAG }}
-          fetch-depth: 0
-      - name: Set env
-        run: |
-          WORKER_DOCKER_TAG=$(echo ${{ env.RELEASE_TAG }} | sed 's/.*\(w.*\)/\1/;s/w/v/')
-          echo "WORKER_DOCKER_TAG=$WORKER_DOCKER_TAG" >> $GITHUB_ENV
+        # with:
+        # ref: ${{ env.RELEASE_TAG }}
+        # fetch-depth: 0
```

Comment:
I'll return it back once build release works fine

Reply:
https://github.com/litentry/litentry-parachain/actions/runs/7669315590
looks like it works fine